packages/-build-infra/src/utilities/test-framework-detector.js

```diff
@@ -15,21 +15,21 @@ module.exports = function(blueprint) {
     let dependencies = this.project.dependencies();
 
     if ('ember-qunit' in dependencies) {
-      if (fs.existsSync(this.path + '/qunit-rfc-232-files')) {
+      if (fs.existsSync(blueprint.blueprintPath + '/qunit-rfc-232-files')) {
         type = 'qunit-rfc-232';
       } else {
         type = 'qunit';
       }
     } else if ('ember-cli-qunit' in dependencies) {
       let checker = new VersionChecker(this.project);
-      if (fs.existsSync(this.path + '/qunit-rfc-232-files') && checker.for('ember-cli-qunit', 'npm').gte('4.2.0')) {
+      if (fs.existsSync(blueprint.blueprintPath + '/qunit-rfc-232-files') && checker.for('ember-cli-qunit', 'npm').gte('4.2.0')) {
```

Comment:
I think these need to be reformatted per the [linting](https://dev.azure.com/ember-data/emberjs/_build/results?buildId=772&view=logs&jobId=3a559e2a-952e-58d2-b8db-2e604a9266d7) errors.
You can easily do this by invoking:
```js
./node_modules/.bin/prettier --write packages/-build-infra/src/utilities/test-framework-detector.js
```

Reply:
I fixed just now using `eslint --fix --plugin prettier .`